api/authentication.yml

```diff
@@ -328,6 +338,34 @@ paths:
           $ref: "#/components/responses/Unauthorized"
         default:
           $ref: "#/components/responses/ServerError"
+
+  /auth/external/login:
+    post:
+      tags:
+        - auth
+        - external
+      operationId: externalPrincipalLogin
+      summary: perform a login using an external authenticator
+      security: [ ]
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/IdentityToken"
```

Comment:
see comment in the `swagger` - it should be an object that one of its fields will be the identity token.